pkg/version/version.go

```diff
@@ -35,3 +43,35 @@ func Get() Info {
 		Platform:   fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
 	}
 }
+
+func GetKubernetesVersion(cluster *v2.Cluster) *KubernetesVersion {
```

Comment:
```suggestion
func GetKubernetesVersion(cluster *v2.Cluster) (*KubernetesVersion, error) {
```
return error if cmd exec fail or version info is mismatch when set debug=true,and print it out.

Do not omit any errors.

Reply:
> return error if cmd exec fail or version info is mismatch when set debug=true,and print it out.
> 
> Do not omit any errors.

but what if `kubectl` or `crictl` binary are missing?

Reply:
After discussion, will show not found and suggest user to start kubernetes cluster using `sealos run`.
<img width="458" alt="image" src="https://user-images.githubusercontent.com/679757/217134570-39aac104-b3bb-4b0d-8183-6b7d4fc7e15e.png">